openapi/src/json-schemas/models/definitions/timestamp.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Timestamp",
+    "type": "string",
+    "pattern": "^(?:[1-9]\\d{3}-(?:(?:0[1-9]|1[0-2])-(?:0[1-9]|1\\d|2[0-8])|(?:0[13-9]|1[0-2])-(?:29|30)|(?:0[13578]|1[02])-31)|(?:[1-9]\\d(?:0[48]|[2468][048]|[13579][26])|(?:[2468][048]|[13579][26])00)-02-29)T(?:[01]\\d|2[0-3]):[0-5]\\d:[0-5]\\d(?:\\.\\d{1,9})?(?:Z|[+-][01]\\d:[0-5]\\d)$"
```

Comment:
JSON Schema already has this baked in e.g. https://github.com/CriminalInjuriesCompensationAuthority/q-templates-application/blob/b40fa7ce73d52c3051acd1b1f1db8fffafa30a61/src/lib/resource/sections/applicant-enter-your-date-of-birth.js#L59

Do we need this regex?

Reply:
I was quite pleased with how complex it was, but no we don't need it! Have changed it to the JSON schema format.